src/mqt/predictor/rl/helper.py

```diff
@@ -38,14 +38,24 @@
     TrivialLayout,
 )
 
-from mqt.predictor import utils
+from mqt.predictor import rl, utils
 
 if sys.version_info < (3, 10, 0):
     import importlib_resources as resources
 else:
     from importlib import resources
 
 
+def compile_prediction(qc: QuantumCircuit, opt_objective="fidelity"):
+    """Returns the compiled quantum circuit with the used combination of compilation options when the original
+    qasm circuit is provided as either a string or a file path.
```

Comment:
As above, that string doesn't really match up with the type information.

Reply:
Adjusted the docu.